pkgs/build-support/make-darwin-bundle/default.nix

```diff
@@ -12,14 +12,14 @@
 
 writeShellScript "make-darwin-bundle-${name}" (''
   function makeDarwinBundlePhase() {
-    mkdir -p "$out/Applications/${name}.app/Contents/MacOS"
-    mkdir -p "$out/Applications/${name}.app/Contents/Resources"
+    mkdir -p "''${!outputBin}/Applications/${name}.app/Contents/MacOS"
```

Comment:
Out of interest, what is the above syntax?  I've not seen that before.

Reply:
It's an indirect reference, the value of the variable is expanded as if it was passed in `${}`. [Fourth paragraph in this section of the Bash reference](https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion). I didn't know about it either, thank you @vcunat IIRC :heart: